doc/man3/OCSP_sendreq_new.pod

```diff
@@ -4,112 +4,116 @@
 
 OCSP_sendreq_new,
 OCSP_sendreq_nbio,
-OCSP_REQ_CTX_free,
-OCSP_set_max_response_length,
-OCSP_REQ_CTX_add1_header,
 OCSP_sendreq_bio,
+OSSL_HTTP_REQ_CTX_free,
+OSSL_HTTP_REQ_CTX_add1_header,
+OSSL_HTTP_REQ_CTX_set_max_response_length,
+OSSL_HTTP_REQ_CTX_i2d,
 OCSP_REQ_CTX_i2d,
+OCSP_REQ_CTX_add1_header,
+OCSP_REQ_CTX_free,
+OCSP_set_max_response_length,
 OCSP_REQ_CTX_set1_req
```

Comment:
Please use consistent order of the functions here, in the synopsis, and in the description.

Reply:
> Please use consistent order of the functions here, in the synopsis, and in the description.

done.